src/core/universal-ad-id.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * The unique creative identifier for this ad.
```

Comment:
There is no "this" ad here. Also, most other descriptions start with a verb, so to be strict, I'd make this:

> Represents a unique identifier for an ad.

Alternatively, the verb is usually either "represents" or "describes" so I guess that makes it redundant. We could also remove it everywhere. I don't have a strong opinion either way but I'd like the docs to be consistent between files.